lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -1324,6 +1357,19 @@ export abstract class CacheManager implements ICacheManager {
         );
     }
 
+    /**
+     * helper to match names
+     * @param entity
+     * @param name
+     * @returns true if the downcased name properties are present and match in the filter and the entity
+     */
+    private matchName(entity: AccountEntity, name: string): boolean {
+        return !!(
+            typeof entity.name === "string" &&
```

Comment:
Can name be something other than string?

Reply:
Refined the check to assume entity.name can only be passed in as string